GenomeFileUtil.spec

```diff
@@ -368,4 +368,37 @@ module GenomeFileUtil {
     funcdef ws_obj_gff_to_metagenome(WsObjGFFToMetagenomeParams params)
                 returns (MetagenomeSaveResult returnVal) authentication required;
 
+    /*
+    Parameters for the update_taxon_assignments function.
+    Fields:
+        workspace_id: a workspace UPA of a Genome object
+        taxon_assignments: an optional mapping of assignments to add or replace. This will perform a
+            merge on the existing assignments. Any new assignments are added, while any existing
+            assignments are replaced.
+        remove_assignments: an optional list of assignment names to remove.
+
+    @optional taxon_assignments remove_assignments
+    */
+    typedef structure {
+        int workspace_id;
```

Comment:
why not have a ref field instead of a workspace and object id

Reply:
Well i don't want the version included. But otherwise :woman_shrugging: